package.json

```diff
@@ -1532,6 +1532,7 @@
     "clean-webpack-plugin": "^3.0.0",
     "cli-progress": "^3.12.0",
     "cli-table3": "^0.6.1",
+    "content-security-policy-parser": "^0.6.0",
```

Comment:
Would you please add this new dependency to `renovate.json` with an appropriate owner?

Reply:
Sure thing!

Reply:
++ thank you!